app/angular/recovery/recovery.html

```diff
@@ -8,36 +8,36 @@
 		/>
 		<section class="recovery-page">
 			<header class="public-logo">
-				<h1>Recuperar senha</h1>
+				<h1>{{ 'Recovery password' | translate }}</h1>
 			</header>
 			<div class="alert" role="alert" data-ng-class="{'hide': !$ctrl.feedback.showing, 'alert-danger': $ctrl.feedback.type=='error', 'alert-success': $ctrl.feedback.type=='success'}">
 				<p>{{ $ctrl.feedback.message }}</p>
 			</div>
 			<form class="recovery-form" name="recoveryForm" ng-submit="$ctrl.submitForm(recoveryForm.$valid)" novalidate>
 				<div class="form-group">
-					<label for="exampleInputEmail1">Email</label>
+					<label for="exampleInputEmail1">{{ 'Email' | translate }}</label>
 					<input
 						type="name"
 						name="name"
 						class="form-control input-lg"
 						id="name"
-						placeholder="Email"
+						placeholder="{{ 'Email' | translate }}"
 						autocomplete="off"
 						required
 						ng-model="$ctrl.mail"
 						ng-class="{'error': recoveryForm.name.$invalid && (!recoveryForm.name.$pristine || $ctrl.submitted)}"
 					/>
 				</div>
-				<button type="submit" class="br-button btn-block">Recuperar</button>
+				<button type="submit" class="br-button btn-block">{{ 'Recovery' | translate }}</button>
 			</form>
 			<aside class="account-options">
 				<div class="navigation">
-					<a href="#" tabindex="1" data-ui-sref="login">Voltar</a>
+					<a href="#" tabindex="1" data-ui-sref="login">{{ 'Back' | translate }}</a>
 				</div>
 			<aside>
 		</section>
 	</div>
-	<a class="br-help hidden-print" href="https://github.com/brmodeloweb/brmodelo-app/issues/new/choose" target="_blank" rel="noopener noreferrer" title="Reportar bug ou requisitar funcionalidade">
+	<a class="br-help hidden-print" href="https://github.com/brmodeloweb/brmodelo-app/issues/new/choose" target="_blank" rel="noopener noreferrer" title="{{ 'Report bug or request functionality' | translate }}">
```

Comment:
```suggestion
	<a class="br-help hidden-print" href="https://github.com/brmodeloweb/brmodelo-app/issues/new/choose" target="_blank" rel="noopener noreferrer" title="{{ 'Report a bug or request a feature' | translate }}">
```

Reply:
In another subject: We should definitively create a component for this bug report thing

Reply:
Work in progress at #266